combat-trainer.lic

```diff
@@ -411,8 +411,9 @@ class LootProcess
     @gem_pouch_adjective = settings.gem_pouch_adjective
     echo("  @gem_pouch_adjective: #{@gem_pouch_adjective}") if $debug_mode_ct
 
-    @loot_delay = 5 # in seconds
+    @loot_delay = 5 || settings.loot_delay # in seconds
```

Comment:
❓ Shouldn't this be `settings.loot_delay || 5`?

Regardless, this should instead be done by adding `loot_delay` to `base.yaml` with a default value of 5.

Reply:
I can do that. My thought was wold not need to add to base.yaml as the either/or would catch.

Reply:
Every setting needs to be in `base.yaml`.